### ISSUE TITLE
Add GSD95 Geoid Model from ca_nrc

### ISSUE THREAD

**Issue by bmckilligan** [OSGeo/PROJ-data#108]
https://webapp.geod.nrcan.gc.ca/geod/data-donnees/geoid.php?locale=en

there are two version nad83(CSRS) and ITRF .byn format

https://webapp.csrs-scrs.nrcan-rncan.gc.ca/geod/process/download-helper.php?file_id=GSD95n83
https://webapp.csrs-scrs.nrcan-rncan.gc.ca/geod/process/download-helper.php?file_id=GSD95

Attached is the ITRF .byn downloaded from nrcan

Additionally, if of use, an exported geotiff created using Qgis. 

[GSD95.zip](https://github.com/OSGeo/PROJ-data/files/12835906/GSD95.zip)
[NGSD95_tiff.zip](https://github.com/OSGeo/PROJ-data/files/12835919/NGSD95_tiff.zip)

**Comment by rouault** [comment]
GSD95 is indicated to be superseded, and it is not referenced in the EPSG dataset. IMHO, it would thus be moderately useful for most end users to include it in PROJ-data. Only advanced ones may use it and they could just fetch the TIFF from the NRCan website.

**Comment by bmckilligan** [comment]
Yes it is deprecated, unfortunately some legacy local coordinate systems use it, the instance I'm looking at is in the petroleum mining sector.  They use if for all their survey work, and have it set up in all the Trimble survey equipment. Ironically the parent companies are part of the IOGP

**Comment by jjimenezshaw** [comment]
@bmckilligan The page clearly says
"GSD95
Superseded model: this geoid model is not compatible with either CGVD2013 or CGVD28."

Which vertical system would you be thinking to use?

**Comment by jjimenezshaw** [comment]
for completeness: https://lists.osgeo.org/pipermail/gdal-dev/2026-July/061704.html
